### DISCUSSION TITLE
Reading event notifications

### DISCUSSION THREAD

**Discussion by andydash1** [HomeSpan/HomeSpan#571] [Q&A] [answered]
Hi,

First - thanks for an excellent piece of work - HomeSpan really is awesome!

Have a newbie question that I can't figure out the answer to.

I have created an outdoor remote temp sensor (based on the RemoteTempSensor example) that sends the temperature data via SpanPoint directly to another device running a full HomeSpan instance (based on the MainDevice example). Works great.

I now would like a third (and possibly fourth etc) device just reading and displaying the outdoor temperature value from HomeKit. (That device has no own sensor, and I would not think it would be needed to receive data direcly from the outdoor unit, rather I would just like to read the outdoor temperature value that now HomeKit currently has stored.) How would I do that?

**Comment by HomeSpan** [comment, marked as answer]
Unfortunately HomeKit does not provide a way for one Accessory to read the values of another Accessory (likely for security reasons).  However, you can use SpanPoint to transmit the temperature from the outdoor sensor to other ESP32s that are not paired to HomeKit, but are using SpanPoint just to receive the temperature from the ESP32 with the temperature sensor.

**Reply by andydash1** [reply]
Got it - thanks.
